CONTRIBUTING.md

```diff
@@ -74,18 +74,11 @@ To run your images locally, execute the following command in the
 project root (after building):
 
 ``` bash
-TAG=latest docker-compose up
+make up
 ```
```

Comment:
yes - this is a very minimal reference to local grapl - it's in `contributing.md` and i think it's _okayish_ to let contributors know they can bring up an environment. Like, consider our new employee Thomas - how would he know about `make up` if not from tribal knowledge?